rust/linux-client/docs/debian_dns.md

```diff
@@ -0,0 +1,52 @@
+# Debian default DNS setups
+
+## Conclusion
+
+As of Feb 16th, the code in Firezone will probably work well on Ubuntu. On Debian we'd need to either ask for `systemd-resolved` to be installed, or fall back on rewriting `/etc/resolv.conf`, since we don't cooperate with NM yet.
```

Comment:
I did a bit of research on this last night. It appears that if you just make sure [this symlink](https://tailscale.com/kb/1188/linux-dns#networkmanager--systemd-resolved) is in place, [NM will use systemd-resolved](https://wiki.archlinux.org/title/systemd-resolved) and then problem solved?

Reply:
We could probably just document that users do that and then we don't have to build a NM integration

Reply:
`systemd-resolved` is not installed by default on Debian, so there's no `/usr/bin/resolvectl`. We agreed in the standup to let that go for now. Ubuntu is one of the most popular desktop distros and they have `resolvectl`. For Debian we can offer to rewrite `/etc/resolv.conf` or later on we can cooperate with NM.